gpg_import.py

```diff
@@ -172,9 +186,10 @@ def _setup_creds(self, key_override=None):
             'check':   '{bin_path} {check_mode} --list-keys {key_id}',
             'delete':  '{bin_path} {check_mode} --batch --yes --delete-keys {key_id}',
             'refresh': '{bin_path} {check_mode} --keyserver {key_id} --keyserver-options timeout={timeout} --refresh-keys {url}',
+            'check-private':  '{bin_path} {check_mode} --list-secret-keys {key_id}',
             'recv':    '{bin_path} {check_mode} --keyserver {key_id} --keyserver-options timeout={timeout} --recv-keys {url}',
             'check-public':  '{bin_path} {check_mode} --list-public-keys {key_id}',
-            'import-public': '{bin_path} {check_mode} --import {key_file}'
+            'import-key': '{bin_path} {check_mode} --import {key_file}'
```

Comment:
I will consider your proposal @tnt. ATM we're giving the module a simple use that might be temporary, in case we need to develop more on the current path I'll give it a try.